python/python_fastapi_server/routers/edr.py

```diff
@@ -420,11 +471,16 @@ def get_params_for_collection(edr_collection: str) -> dict[str, Parameter]:
         else:
             label = param_el["name"]
 
+        symbol = Symbol(value=param_el["unit"], type=SYMBOL_TYPE_URL)
+        param_id = param_el["name"]
+        # If standard name was configured, use that instead with a vocabulary
+        if "standard_name" in param_el:
+            param_id = VOCAB_ENDPOINT_URL + param_el["standard_name"]
         param = Parameter(
             id=param_el["name"],
-            observedProperty=ObservedProperty(id=param_el["name"], label=label),
+            observedProperty=ObservedProperty(id=param_id, label=label),
             type="Parameter",
-            unit=Unit(symbol=param_el["unit"]),
+            unit=Unit(symbol=symbol),
```

Comment:
These are breaking changes. Are the users (e.g. Geoweb) okay with that?

Reply:
Yes, it will be allright since the KDP edr api does the same.